docs/manuals/userman/gui/pages/index.md

```diff
@@ -49,313 +34,225 @@ Pages can be defined using two different techniques:
-         "/": root_md,
-         "page1": page1_md,
-         "page2": page2_md
-       }
-       Gui(pages=pages).run()
-    ```
-    This application does the same as in the previous example, except that you now
-    have the footer line (*'This application was created...'*) in all the pages of
-    your application.
+        with tgb.Page() as page:
+            tgb.text(value="# Taipy Getting Started", mode="md")
+            tgb.text(value="Value: {value}")
+            tgb.slider(value="{value}", on_change=on_slider)
+            tgb.chart(data="{data}")
 
-# Dialogs
 
-Applications sometimes need to prompt the user to indicate a situation or request
-input of some sort. Dialogs are forms that can be displayed on top of the page
-the user is looking at, prompting for some input.
+        if __name__ == "__main__":
+            data = compute_data(value)
 
-To create a dialog, you will use a [`dialog`](../viselements/generic/dialog.md) control in your
-page. The dialog holds a page content or a *Partial* (see [Partials](#partials)).
+            Gui(page=page).run(title="Dynamic chart")
+        ```
 
-You can control whether the dialog is visible or not, and what to do when the end-user
-presses the *Validate* or *Cancel* button, so your application can deal with the
-user's response.
+## Registering a single page
 
-!!! example
+Once you have created an instance of a page renderer for a specific piece of text or Python code,
+you can register that page to the `Gui^` instance used by your application.
 
-    Here is an example of how you would create a dialog, directly in your Markdown
-    content:
+The `Gui^` constructor can accept the raw content of a page as Markdown text, a Page object and 
+create a new page for you. That would be the easier way to create applications that have a 
+single page. Here is how you can create and register a page, defined as Markdown content, in a 
+Taipy application:
 
+=== "page variable"
     ```python
-       ...
-       page="""
-       ...
-       <|{dialog_is_visible}|dialog|
-       Enter a name:
-
-       <|{name}|input|>
-       |>
-       ...
-       """
+    from taipy import Gui
 
-       Gui(page).run()
-    ```
+    page = "# First page"
 
-Please refer to the documentation page on the [`dialog`](../viselements/generic/dialog.md)
-control for more details and examples.
+    Gui(page=page).run()
+    ```
+=== "add_page function"
+    ```python
+    from taipy import Gui
+    import taipy.gui.builder as tgb
 
-# Partials
+    with tgb.Page() as page:
+        tgb.text("# First Page", mode="md")
 
-There are page fragments that you may want to repeat on different pages. In that situation,
-you will want to use the *Partial* concept: a *Partial* is similar to a page (and built
-in a very similar way) that can be used multiple times in different visual elements.
-This prevents you from having to repeat yourself when creating your user interfaces.
+    Gui()
+    gui.add_page(page)
```

Comment:
Remove this line and the next

Reply:
I am not sure to understand this. I am showing the two ways to add pages. I fixed the code btw

Reply:
I see no point (at all) to split `Gui(page).run()` into three lines.
And homogeneity is important.